src/common/html-attributes/index.js

```diff
@@ -1,5 +1,5 @@
 const assign = require('core-js-pure/features/object/assign');
-
+const skipAttributes = ['*', 'htmlAttributes', 'renderBody', 'widgetState', '__widgetProps', '___widgetProps'];
```

Comment:
For `___widgetProps` we'll a actually need to do a release of Marko 4 to make this non enumerable since the above code would actually not work when running against the production version of Marko since the `___` properties get minified.

Reply:
That's fine, but I think it should be safe to leave it in there until the release.

Reply:
The thing is it would give a false sense of security. It works in dev, but would break in prod (for Marko 4 apps w/ compatibility layer). We shouldn't release it like this.

Reply:
I can try to get out a release of Marko today along side this. We should make sure to include in the release notes that the latest version of Marko is required if using with Marko 4.

Reply:
And marko v3 too correct? Since this should exist there too right? Or just marko 4?

Reply:
The `___` properties are only there in Marko 4, and get renamed/minified in prod.

Reply:
I just realized in Marko 3 this list is missing `widgetBody`

Reply:
Actually `widgetProps` as well was there in Marko 3. Not sure why we made it a `___` prop in Marko 4. I am going to update the compatibility layer in Marko 4 to actually just use the bare `widgetProps` name as well.

Reply:
So should I keep the __widgetProps (two underscore) OR rename to widgetProps

Reply:
Anyways, I added both widgetProps and widgetBody then. Kept the two underscore __widgetProps

Reply:
The full list should be `['htmlAttributes', 'renderBody', 'widgetState', 'widgetProps', 'widgetConfig', 'widgetBody']`.

We shouldn't need `*` either because we changed everything to `targetProperty: null`.

Reply:
👍 Should be updated